myhpi/search/templates/search/search_result.html

```diff
@@ -9,7 +9,7 @@ <h5 class="text-break">{{ result.title }}</h5>
                 {{ result.specific.date|date:"SHORT_DATE_FORMAT" }}
             </time>
         </hgroup>
-        <span class="text-dark">
+        <span class="search-preview">
             {% if result.search_description %}
                 {{ result.search_description }}
```

Comment:
The input elements now seems to be gray by default, which looks like they would be disabled. Could we make them white again?

![grafik](https://github.com/user-attachments/assets/62603688-e89a-42b0-ba5a-82b77b5bfd1d)

![grafik](https://github.com/user-attachments/assets/00105150-61fb-4201-b888-352cc47912c9)